devops/modules/2_cluster-configuration/task_3.md

```diff
@@ -57,20 +53,15 @@ In this task, you will configure and deploy a Kubernetes (K8s) cluster on AWS us
    - Terraform code is created or extended to manage AWS resources required for the cluster creation.
    - The code includes the creation of a bastion host.
 
-2. **Cluster Deployment (60 points)**
-
-   - A K8s cluster is deployed using either kOps or k3s.
-   - The deployment method is chosen based on the user's preference and understanding of the trade-offs.
-
-3. **Cluster Verification (10 points)**
+2. **Cluster Verification (50 points)**
 
    - The cluster is verified by running the `kubectl get nodes` command from the local computer.
```

Comment:
local computer or bastion?

Reply:
Moved "...from local computer" to the additional tasks.